crates/forge/src/scarb/config.rs

```diff
@@ -20,8 +20,10 @@ pub struct ForgeConfigFromScarb {
     pub detailed_resources: bool,
     /// Save execution traces of all test which have passed and are not fuzz tests
     pub save_trace_data: bool,
-    /// Builds profile of all test which have passed and are not fuzz tests
+    /// Build profiles of all tests which have passed and are not fuzz tests
```

Comment:
```suggestion
    /// Build profile data of all tests which have passed and are not fuzz tests
```

Profiles seems weird in this context to me

Reply:
Sorry for the confusion. The description was modified purely for the sake of consistency. It is a description for the `--build-profile` flag and is not really related to this PR.